src/app/tasks-board/new-task/new-task.component.html

```diff
@@ -1,12 +1,12 @@
-<div class="new-task">
-  <input [(ngModel)]="name" type="text" placeholder="Enter title">
+<form class="new-task" name="new-task" (submit)="onSave(); $event.preventDefault()">
```

Comment:
awesome! not sure why I hadn't use form at first place :) will add validations as well..